content/docs/api-reference/iam-subjects-api/index.md

```diff
@@ -0,0 +1,205 @@
+     -H "@$BEARER_TOKEN_FILE" \
+     $URL/archivist/iam/v1/subjects/6a951b62-0a26-4c22-a886-1082297b063b
+```
+
+#### Fetch IAM Subjects by name (v1)
+
+To fetch all IAM subjects with a specific name, `GET` the `/subjects` resource and filter on `display_name`:
+
+```bash
+curl -g -v -X GET \
+     -H "@$BEARER_TOKEN_FILE" \
+     $URL/archivist/iam/v1/subjects?display_name=Acme
+```
+
+Each of these calls returns a list of matching IAM subjects records in the form:
+
+```json
+{
+    "subjects": [
+        {
+            "identity": "subjects/6a951b62-0a26-4c22-a886-1082297b063b",
+            "display_name": "Some description",
+            "wallet_pub_key": ["key1"],
+            "wallet_address": ["address1"],
+            "tessera_pub_key": ["key2"]
+        },
+        {
+            "identity": "subjects/12345678-0a26-4c22-a886-1082297b063b",
+            "display_name": "Some otherdescription",
+            "wallet_pub_key": ["key5"],
+            "wallet_address": ["address5"],
+            "tessera_pub_key": ["key7"]
+        }
+    ]
+}
+```
+
+### IAM Subject Deletion
+
+To delete an IAM subject, issue the following request:
+
+```bash
+curl -v -X DELETE \
+    -H "@$BEARER_TOKEN_FILE" \
+    -H "Content-type: application/json" \
+    $URL/archivist/iam/v1/subjects/47b58286-ff0f-11e9-8f0b-362b9e155667
+```
+
+The response is `{}`.
+
+### IAM Subject Update
+
+Define the subjects parameters to be changed and store in `/path/to/jsonfile`:
+
+```json
+{
+    "wallet_pub_key": ["key1"],
+    "tessera_pub_key": ["key2"]
+}
+```
+
+Update the IAM Subject:
+
+```bash
+curl -v -X PATCH \
+    -H "@$BEARER_TOKEN_FILE" \
+    -H "Content-type: application/json" \
+    -d "@/path/to/jsonfile" \
+    $URL/archivist/iam/v1/subjects/47b58286-ff0f-11e9-8f0b-362b9e155667
+```
+
+The response is:
+
+```json
+{
+    "identity": "subjects/3f5be24f-fd1b-40e2-af35-ec7c14c74d53",
+    "display_name": "Some description",
+    "wallet_pub_key": ["key1"],
+    "wallet_address": ["address1"],
+    "tessera_pub_key": ["key3"]
+}
+```
+
+### IAM Subject Self Entry
+
+There is an immutable entry in the subjects API called `Self` that contains the keys for the hosting organisation of the RKVST Tenant.
+
+This entry cannot be deleted or updated.
+
+This special identity is:
+
+```bash
+subjects/00000000-0000-0000-0000-000000000000
+```
+
+#### Fetch self IAM Subject by identity (v1)
+
+```bash
+curl -v -X GET \
+     -H "@$BEARER_TOKEN_FILE" \
+     $URL/archivist/iam/v1/subjects/00000000-0000-0000-0000-000000000000
+```
+
+The response is:
+
+```json
+[
+    {
+        "identity": "subjects/00000000-0000-0000-0000-000000000000",
+        "display_name": "Some description",
+        "wallet_pub_key": ["key1"],
+        "wallet_address": ["address1"],
+        "tessera_pub_key": ["key3"]
+    }
+]
+```
+
+## IAM Subjects OpenAPI Docs
+
+{{< openapi url="https://raw.githubusercontent.com/jitsuin-inc/archivist-docs/master/doc/openapi/subjects.swagger.json" >}}
```

Comment:
Waiting for fix from Engineering on the title of the OpenAPI spec

https://github.com/jitsuin-inc/archivist-docs/blob/acc38aba651888b0a8469505a71c77e1e43f078f/doc/openapi/subjects.swagger.json#L4

Does not fit our standard "<name of api> API" format (e.g. https://github.com/jitsuin-inc/archivist-docs/blob/acc38aba651888b0a8469505a71c77e1e43f078f/doc/openapi/tenancies.swagger.json#L4)